copyspecial.py

```diff
@@ -16,29 +16,60 @@
 import argparse
 
 # This is to help coaches and graders identify student assignments
-__author__ = "???"
+__author__ = "knmarvel with madarp"
 
```

Comment:
does this run on both python2 and python3?  if not, maybe put the guard code in...